src/components/select.css

```diff
@@ -0,0 +1,20 @@
+.modal-content {
+
+  /* border:solid tomato; */
+margin: 0rem auto;
+
+}
+
+img {
+
+width:350px;
+height:350px;
+
```

Comment:
So far really nice work on this, you are almost there! 👍  Just need a bit of tweaking on your images and the modal display.

Feedback here: 
Images are looking great but don't fit the container. To fix this, set a `width: 100%`